README.md

```diff
@@ -1,2 +1,3 @@
 # hello-world
 the first chance to try the Github
```

Comment:
:+1: this PR looks great - it's ready to merge! :smile: 
![timg](https://user-images.githubusercontent.com/31680196/30098456-a5f85080-9314-11e7-9137-2aed1c3f447e.gif)